### DISCUSSION TITLE
How to correctly setup MDC?

### DISCUSSION THREAD

**Discussion by jarlehansen** [ExpediaGroup/graphql-kotlin#1347] [Q&A] [answered]
I am having an issue with the MDC info being empty when inside the query. Any examples on how I can get the MDC values passed on into the execution of the query?

**Comment by jarlehansen** [comment, marked as answer]
Only way I have found is to override `FunctionDataFetcher` and insert the `MDCContext` here in the constructor.
Is there an easier way.

**Reply by dariuszkuc** [reply]
https://github.com/ExpediaGroup/graphql-kotlin/pull/1349 introduces coroutine context propagation (will be available in v6). In older/current versions of the lib you have to explicitly provide it in the function data fetcher.
